src/main/java/edu/harvard/h2ms/seeders/RoleSeeder.java

```diff
@@ -0,0 +1,41 @@
+package edu.harvard.h2ms.seeders;
+
+import static java.util.Arrays.asList;
+
+import edu.harvard.h2ms.domain.core.Role;
+import edu.harvard.h2ms.repository.RoleRepository;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.event.ContextRefreshedEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+
+@Component
+public class RoleSeeder {
+
+  private RoleRepository roleRepository;
+
+  @Autowired
+  public RoleSeeder(RoleRepository roleRepository) {
+    this.roleRepository = roleRepository;
+  }
+
+  @EventListener
+  public void seed(ContextRefreshedEvent event) {
+    seedRoleTable();
+  }
+
+  private void seedRoleTable() {
+    if (roleRepository.count() == 0) {
+      List<List<String>> records = asList(asList("ADMIN"), asList("USER"), asList("OBSERVER"));
```

Comment:
Why is this a nested list and not just this:

```java
List<String> = asList("ADMIN", "USER", "OBSERVER")
```

Reply:
Should Observer be a special role? If a user is not an Admin and not an Observer, then what can they really even do in the app?

Reply:
I updated the list to be un-nested.

Correct. Observer is a special role so that only they can create events. User is just the default role until an admin changes it if they choose to.